notes-webapp/src/main/webapp/vue-app/notes/components/NoteTreeviewDrawer.vue

```diff
@@ -709,6 +709,12 @@ export default {
       }
       return itemsArray;
     },
+    naturalSort(items) {
```

Comment:
can we get the drafts sorted directly from back without sorting them in front ?

Reply:
It is used only in case of drafts, i don't see a reason to not do that in the front since i cant do it in request